File: code24.py
res = 0
s = [2,3,5,7,11,13,17,19]
for i in range(10,100):
    v = str(i)
    
    f = 1
    for j in range(0,len(v)-1):
        if (int(v[j])+int(v[j+1])) in s:
            f=0
            break
    if f==1:
        print(v)
        res+=1
print(res)

File: code24.java
public class code24 {
    public static boolean checkBit(int value,int index){
        int mask = 1<<index;
        return (value&mask)>0;
    }
    public static  boolean checkBitOne(int value,int index){
        int mask = 1<<index;
        return (value&mask)==0;
    } 
    public static int countLastSet(int value){
        int res = 0;
        int index = 7;
        while(index>-1){
            if(checkBit(value,index)){
                res+=1;
                index-=1;
            }
            else{
                return res;
            }
        }
        return res;
    }
    public static void main(String[] args) {
        // System.out.println(checkBit(3, 0));
        // System.out.println(countLastSet(197));
        // System.err.println(checkBitOne(4, 2));
        
    }
}